### PR TITLE
Rs 2022 07 30.1

### PR DESCRIPTION
**Pull request by rschlaikjer**
2022-07-30-211202_1920x1140_scrot

### CODE REVIEW

File: script/Ciel/Day 5/05_01_CIEL05_1D.txt [hunk 1]
```diff
@@ -1,48 +1,54 @@
...
-They only move actively when they perceive their prey. Their range is not that wide, maybe ten meters at most.
+They only spring into action when they can perceive their prey,
+ and it seems like they can only sense things that are fairly close by.
+ Probably only ten meters or so at most.
 }
 [sha:70198b77071dae624924dd814aac8b54f349d16f]{
 -- Page 149, Offset 11288.
 -- ……知覚する方法が視力によるものなのか、聴覚によるものなのか、あるいは別の何かなのかは、アレになってみなければ判らない話ではあるが。
-...Whether they perceive by sight, hearing, or something else, I wouldn't know unless I turned into one.
+I don't know whether they perceive things based on sight, hearing, or something else entirely,
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_1D.txt:27]: i think it sounds better with "hearing" first
hearing, sight, or something else entirely.

File: script/Ciel/Day 5/05_01_CIEL05_1D.txt [hunk 2]
```diff
@@ -123,57 +147,65 @@ A %{i}normal%{/i} human would go insane! There's no way you could be used to see
...
 -- Page 162, Offset 11310.
 -- あり得ない空間。
-An impossible space.
+This is an unimaginable space.
 }
 [sha:695fe34d82a2bca1f4bc25c239843b03dca5ecac]{
 -- Page 162, Offset 11311.
 -- 蟻の巣のように張り巡らされた地下墓地。動く死体たち。
-An underground graveyard stretching out like an ant nest. Walking dead bodies.
+A sprawling underground graveyard that resembles some kind of ant nest.
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_1D.txt:166]: i believe they're called ant colonies

File: script/Ciel/Day 5/05_01_CIEL05_1D.txt [hunk 3]
```diff
@@ -123,57 +147,65 @@ A %{i}normal%{/i} human would go insane! There's no way you could be used to see
...
 -- Page 163, Offset 11316.
 -- 理性ある人間だからこそ、この中で<正気|じぶん>を保つ事はできない。
-It is precisely because humans are rational, that they cannot maintain their sense of self here.
+It is precisely because humans are rational that they cannot maintain their sense of self here.
 }
 [sha:cadf1aa963ab5491f9eabc43b322d12ad8568218]{
 -- Page 163, Offset 11317.
 -- まっとうな人間であるのなら。ここでは皆、<死体|やつら>以上に<知性|あたま>のない動物になるしかない。
-Proper human beings, in this kind of place, have no other choice but to become animals no more intelligent than %{i}them%{/i}.
+Normal human beings, at any rate.
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_1D.txt:199]: no comma

File: script/Ciel/Day 5/05_01_CIEL05_1D.txt [hunk 4]
```diff
@@ -258,37 +297,42 @@ That's something a good-for-nothing like me never managed to do.
...
 -- 本人は気がついていないようだけど、もうパニックを起こしていない。
-It seems like she hasn't noticed herself that she isn't panicking anymore.
+It seems she's stopped panicking, even if she herself hasn't noticed it yet.
 }
 [sha:0bcdc094f72326aa972a1e6255cd369f9daa2c93]{
 -- Page 171, Offset 11339.
 -- 「じゃあそのたまたまを続けましょう。今は俺と先生、二人もいるんです。助け合えばなんとか―――」
-"Then, let's continue with that luck. Now that there's two of us, you and me. If we help each other, we'll―――"
+"Alright, let's just keep being lucky then.
+ Plus, there are two of us now, you and me.
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_1D.txt:324]: now: you and me

File: script/Ciel/Day 5/05_01_CIEL05_1D.txt [hunk 5]
```diff
@@ -393,12 +445,16 @@ In the first place, what did I come here for?
...
+ D-Don't tell me you plan on going back?!
+ Back %{i}down%{/i}?!
+ No, don't do it!
+ There weren't just ten or twenty of those zombies back there!
 }
 [sha:6eb315f5f1cbfe0a6cdefe4c693a2aa3bfbf0dd6]{
 -- Page 182, Offset 11363.
 -- ううん、それより―――もう間に合わないし、そもそも、助ける算段とかあるの!?」
-No, more importantly―――You won't make it in time, and anyway, do you even have a way to save them?!"
+No, more importantly―――you won't make it in time, and do you even %{i}have%{/i} a way to save her?!"
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_1D.txt:457]: no italics

File: script/Ciel/Day 5/05_01_CIEL05_1D.txt [hunk 6]
```diff
@@ -418,37 +474,42 @@ I take a deep breath and calm myself down.
...
 -- いま彼女は“下”と言った。その広場とやらは、この通路の行き着く先にある可能性が高い。
-She just said 'down there'. There's a good chance that I'll find that large room when I reach the end of it.
+She said 'down' just a second ago.
+ It sounds like there's a good chance I'll find that large room if I keep heading down this passage.
 }
 [sha:4af48fafb1ff772df7a86cb5255abe62f941b83a]{
 -- Page 185, Offset 11369.
 -- 「い、行くの!?　わ、わたしは行かない、行かないわよ!?」
-"Y-You're going!? I-I won't go, I won't go!?"
+"Y-You're going!? I-I won't go, I won't go you hear me!?"
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_1D.txt:489]: go, you hear me

File: script/Ciel/Day 5/05_01_CIEL05_1D.txt [hunk 7]
```diff
@@ -483,76 +547,84 @@ Aside from me, a single weapon like that can't possibly drive back the Dead. Sti
...
 -- Page 195, Offset 11387.
 -- ……俺の判断にノエル先生を巻きこんでしまった。
-...I dragged Noel-sensei into what I decided to do.
+I've ended up dragging Noel-sensei into what I set out to do...
 }
 [sha:b1fde65e78deb72eeed50ae08f9732756b9d3f6c]{
 -- Page 195, Offset 11388.
 -- せめて、彼女が恐怖で圧し潰されないように、ぎゅっと固く手を握る。
-At the very least, so that she won't be overwhelmed by fear, I tightly grip her hand.
+I hold her hand tightly, so that at least she won't be overwhelmed by fear.
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_1D.txt:598]: drop comma

**Review by Avuxo — APPROVED** [review]
That's right...

File: script/Ciel/Day 5/05_01_CIEL05_1D.txt (hunk 2 above)
Comment (rschlaikjer) [reply script/Ciel/Day 5/05_01_CIEL05_1D.txt:166]: Funnily enough that's what I thought too but I actually googled it to be sure [(wiki)](https://en.wikipedia.org/wiki/Ant_colony#Names)
> The term "ant colony" refers to a population of workers, reproductive individuals, and brood that live together, cooperate, and treat one another non-aggressively. [...] "Ant nests" are the physical spaces in which the ants live. These can be underground, in trees, under rocks, or even inside a single acorn.[[6]]

File: script/Ciel/Day 5/05_01_CIEL05_1D.txt (hunk 3 above)
Comment (rschlaikjer) [reply script/Ciel/Day 5/05_01_CIEL05_1D.txt:199]: Hmm, I kinda like the pause here for emphasis.

File: script/Ciel/Day 5/05_01_CIEL05_1D.txt [hunk 8]
```diff
@@ -123,57 +147,65 @@ A %{i}normal%{/i} human would go insane! There's no way you could be used to see
...
 -- 理性ある人間だからこそ、この中で<正気|じぶん>を保つ事はできない。
-It is precisely because humans are rational, that they cannot maintain their sense of self here.
+It is precisely because humans are rational that they cannot maintain their sense of self here.
 }
 [sha:cadf1aa963ab5491f9eabc43b322d12ad8568218]{
 -- Page 163, Offset 11317.
 -- まっとうな人間であるのなら。ここでは皆、<死体|やつら>以上に<知性|あたま>のない動物になるしかない。
-Proper human beings, in this kind of place, have no other choice but to become animals no more intelligent than %{i}them%{/i}.
+Normal human beings, at any rate.
+ Every ordinary human dropped into this place would be doomed to lose their heads,
```
Comment (yk1512) [inline script/Ciel/Day 5/05_01_CIEL05_1D.txt:200]: Any instead of every

File: script/Ciel/Day 5/05_01_CIEL05_1D.txt (hunk 3 above)
Comment (yk1512) [reply script/Ciel/Day 5/05_01_CIEL05_1D.txt:199]: comma is fine for me

File: script/Ciel/Day 5/05_01_CIEL05_1D.txt [hunk 9]
```diff
@@ -258,37 +297,42 @@ That's something a good-for-nothing like me never managed to do.
...
-We're going to be killed here...! We were saved by luck just now, but we're going to be killed all the same!"
+We're going to be killed down here...!
+ We got saved by a stroke of luck back then,
+ but we're still going to end up killed!"
 }
 [sha:0e043066266734535072006f38e7b5891dfdbc09]{
 -- Page 170, Offset 11337.
 -- ノエル先生は涙目で訴えてくるが、その眼には先ほどあった憎しみが消えていた。
-Noel-sensei complains with tears in her eyes, but the hatred that had been there earlier has disappeared.
+Noel-sense has tears in her eyes as she wails,
```
Comment (yk1512) [inline script/Ciel/Day 5/05_01_CIEL05_1D.txt:312]: sensei

File: script/Ciel/Day 5/05_01_CIEL05_1D.txt [hunk 10]
```diff
@@ -483,76 +547,84 @@ Aside from me, a single weapon like that can't possibly drive back the Dead. Sti
...
 -- Page 191, Offset 11380.
 -- 先生にナイフを手渡して、足を進ませる。
-I hand over my knife to Sensei and walk away.
+I hand her my knife and begin to walk away.
 }
 [sha:58f41365b846fd13299f7dd208bcef0b10f9ad04]{
 -- Page 191, Offset 11381.
 -- ……申し訳ないけど、これが今の俺にできる精一杯だ。
-...I'm sorry, but this is the best I can do right now.
+It's a shame, but... this is the best I manage to do right now.
```
Comment (yk1512) [inline script/Ciel/Day 5/05_01_CIEL05_1D.txt:555]: I prefer the 'can do', or 'can manage to do'

**Review by yk1512 — APPROVED** [review]

File: script/Ciel/Day 5/05_01_CIEL05_1D.txt (hunk 3 above)
Comment (Avuxo) [reply script/Ciel/Day 5/05_01_CIEL05_1D.txt:199]: I'd still rather it drop but I like democracy more than commas so it can stay

**Pull request merged**